include/hpdf_config.h.cmake

```diff
@@ -40,13 +40,6 @@
 /* debug trace enabled */
 #cmakedefine LIBHPDF_DEBUG_TRACE
 
-/* Version numbers */
-#define HPDF_MAJOR_VERSION "@PROJECT_VERSION_MAJOR@"
-#define HPDF_MINOR_VERSION "@PROJECT_VERSION_MINOR@"
-#define HPDF_BUGFIX_VERSION "@PROJECT_VERSION_PATCH@"
```

Comment:
you should drop the quotes for HPDF_MAJOR_VERSION/HPDF_MINOR_VERSION/HPDF_BUGFIX_VERSION
vtk detection assumes they are integers

Reply:
For reference, the old `hpdf_version.h` file: https://github.com/libharu/libharu/blob/fdd021c79afaea8b2ae1a1141978ef7265869bed/include/hpdf_version.h#L3-L8

Reply:
so HPDF_VERSION_ID too